components/Layout/Header.jsx

```diff
@@ -5,9 +5,9 @@ import LinkItem from '../ui/LinkItem'
 const Header = () => {
   return (
     <Wrapper>
-        <div className='py-12 flex-col lg:flex-row gap-5 flex text-xl sm:text-2xl justify-between -tracking-tighter sm:tracking-normal'>
+        <div className='py-12 flex-col lg:flex-row gap-5 flex text-xl sm:text-2xl justify-between -tracking-tighter'>
             <div>Ape Unit</div>
-            <div className="max-w-189.25 xl:max-w-201.5">Unit➇ is a pioneering technology company specialing in decentralised technologies that creates end-to-end digital experiences for protocols including <Link href="#" className='text-purple'>Ethereum,</Link> <Link href="#" className='text-purple'>Tezos,</Link> <Link href="#" className='text-purple'>Near,</Link> <Link href="#" className='text-purple'>Algorand,</Link> <Link href="#" className='text-purple'>Celo</Link> and more. Its team of 2,500+ digital specialists across 30+ locations on 5 continents delivers pioneering work on a global scale with a boutique culture. <LinkItem>Contact us</LinkItem></div>
+            <div className="max-w-189.25 xl:max-w-201.5">Unit➇ is a pioneering technology company specialing in decentralised technologies that creates end-to-end digital experiences for protocols including <span className='text-purple'>Ethereum,</span> <span className='text-purple'>Tezos,</span> <span className='text-purple'>Near,</span> <span className='text-purple'>Algorand,</span> <span className='text-purple'>Celo</span> and more. Its team of 2,500+ digital specialists across 30+ locations on 5 continents delivers pioneering work on a global scale with a boutique culture. <LinkItem underlined>Contact us</LinkItem></div>
```

Comment:
Instead of using many span tags to put in the purple words, it would be better to use only one span tag since they all have the same styles.